TicTacToe/TicTacToe/Board.cs

```diff
@@ -1,18 +1,38 @@
 ﻿using System;
+using System.Collections.Generic;
 
 namespace TicTacToe {
     
     public class Board {
 
+        private List<string> availableSpaces = new List<string>();
```

Comment:
Are you caching available spaces for performance reasons, or is this just the implementation you chose at this point?